cli/client.go

```diff
@@ -1975,6 +1976,32 @@ var clientCancelTransfer = &cli.Command{
 	},
 }
 
+var clientCancelRetrievalDeal = &cli.Command{
+	Name:  "cancel-retrieval-deal",
```

Comment:
```suggestion
	Name:  "cancel-retrieval",
```

Reply:
Or maybe even make this a sub-command of the `retrieve` command -- 
* `lotus client cancel-retrieval-deal` vs
* `lotus client cancel-retrieval` vs
* `lotus client retrieve cancel`

Reply:
I like `lotus client retrieve cancel` 👍

Reply:
Agree.